examples/config.kinesis.minimal.hocon

```diff
@@ -0,0 +1,30 @@
+# The minimum required config options for loading from kinesis
```

Comment:
Can I suggest to rename `examples` dir into `config`. All other apps have configs in that that dir and I like it.

Reply:
Also something very stylistic - can you make it even more JSON-like, surrounding top-level with braces?